commcare_connect/program/tables.py

```diff
@@ -30,6 +30,8 @@ def render_budget(self, record):
         return f"{record.program.budget} {record.program.currency}"
 
     def render_manage(self, record):
+        if record.status == ProgramApplicationStatus.ACCEPTED:
```

Comment:
According to [line](https://github.com/dimagi/commcare-connect/pull/389/files#diff-1914cafdbf59040f7f77df353660e2634bcf3ec8e789b018d9864bc6ee7fded2R138) now even when NM decline the request the "Apply" and "Decline" will still appear.